README.md

```diff
@@ -67,19 +67,29 @@ cli/lightning-cli newaddr
 
 # Returns a transaction id <txid>
 bitcoin-cli -testnet sendtoaddress <address> <amount>
+```
 
-# Retrieves the raw transaction <rawtx>
-bitcoin-cli -testnet getrawtransaction <txid>
+Eventually `lightningd` will be able to receive the funds, it just
+requires `lightningd` catching up to the tip and confirming it on a
+block.
```

Comment:
Maybe just write:

    `lightningd` will register the funds once the transaction is confirmed.